utils/httpUtils.ts

```diff
@@ -40,15 +60,35 @@ export const get = async (url: string, args: any, context: any) => {
 export const getFullResponse = async (url: string, args: any, context: any) => {
   try {
     const baseURL = process.env.API_URL;
-    const urlPrefix = args.snapshotLinkId ? `/pub/${args.snapshotLinkId}` : "";
-    const response = await fetch(baseURL + urlPrefix + url, {
-      method: "GET",
-      headers: {
-        Cookie: context.request.headers.get("cookie"),
-        "Content-Type": "application/json",
-      },
-    });
-    return response;
+    const nextGenEnabled = await isNextGenEnabled(context);
+    if (nextGenEnabled) {
+      // next gen details
+      const czidServicesToken = await getEnrichedToken(context);
+      const query = context.params.query;
+      const response = await fetch(process.env.API_URL + url, {
```

Comment:
This is not the correct URL for nextGen, because this is concatenating the Rails base URL with the endpoint passed in, right?  Since we don't actually check the the flag until we get to this method, the resolvers won't actually know which URL to call.  Though for calling nextGen, since it's graphQL, I don't think we have to. worry about a different URL, it's always going to be `/graphql`.

Reply:
sorry, oversight. Fixed.